install/Knative-with-EKS.md

```diff
@@ -0,0 +1,105 @@
+# Knative Install on Elastic Container Service for Kubernetes (Amazon EKS)
+
+This guide walks you through the installation of the latest version of Knative
+using pre-built images.
+
+## Before you begin
+
+Knative requires a Kubernetes cluster v1.11 or newer. This guide walks you
+through creating a cluster with the correct specifications for Knative on Elastic Container Service for Kubernetes (Amazon EKS).
+
+
+## Creating a Kubernetes cluster
+
+Follow the instruction on: https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html
```

Comment:
The page you linked to has a lot going on, and I don't think all of it is necessary for someone to get started with Knative. I'd actually recommend pulling out the key steps to get the EKS cluster up and running and moving them into this guide. That's the value of the platform specific guides; they're intended to be a complete walk through of the steps necessary to get started with Knative. Compare this guide to the existing GKE guide -- it covers every config step necessary for creating a cluster on Google Cloud; that's what I think we should aim for with EKS.

Reply:
GKE: https://github.com/knative/docs/blob/master/install/Knative-with-GKE.md

Reply:
Done